src/helpers/localStorageHelpers.js

```diff
@@ -81,17 +81,16 @@ export const removeFavoriteCategory = (category) => {
   return true;
 };
 
-export const getDrafts = () => {
-  return store.get('drafts') || {};
-};
+export const getLayout = () =>
+  store.get('layout') || 'card';
```

Comment:
Maybe a "settings" state root object would make more sense. We could include favorites + bookmarks + locale + layout + drafts data on that. What your throughs about this?